ServerCore/wwwroot/css/pd2020/styles.css

```diff
@@ -0,0 +1,105 @@
+/* For logo*/
```

Comment:
Not sure whether this CSS will work or not. The PD2019 one works because of some code inserted into bundleconfig.json. Not sure whether it's necessary to do a similar trick for this version; let me see if I can figure out how to check out this branch and build it...

Reply:
Yeah, the CSS is not working. Puttering...